src/render/View3D.cpp

```diff
@@ -212,7 +212,7 @@ void View3D::centerOnGeometry(const QModelIndex& index)
     m_cursorPos = geom.centroid();
     m_camera.setCenter(m_cursorPos);
     double diag = (geom.boundingBox().max - geom.boundingBox().min).length();
-    m_camera.setEyeToCenterDistance(diag*0.7);
+    m_camera.setEyeToCenterDistance(diag*0.7 + 0.01);
```

Comment:
Thanks for fixing this!  There's no obvious choice for the epsilon, I guess we'll go with this and see what happens :-)